src/FiniteElement.m

```diff
@@ -563,10 +563,10 @@ function forwardSimulation(obj, ocp, Uk, s_sot, p_stage)
 
             for j = 1:dims.n_s
                 % Multiply by s_sot_k which is 1 if not using speed of time variable
-                [fj, qj] = model.f_x_fun(X_ki{j}, obj.rkStageZ(j), Uk, p_stage);
+                [fj, qj] = model.f_x_fun(X_ki{j}, obj.rkStageZ(j), Uk, p_stage, model.v_global);
```

Comment:
why is `v_global` taken out of `model`, but not out of the local workspace (comming e.g. from unfold struct) like the other variables?

Reply:
I'd like to _eventually_ tear out all (or almost all) of the `unfold_struct` usage. Inside the problem definition classes there is already none.

Reply:
This is s good point actually.
Recently, I discussed with @FreyJo, and we agreed that actually having the prefix to where a function/variable belongs, improves the code readability.